package-lock.json

```diff
@@ -1,8 +1,8813 @@
 {
   "name": "mcss-website-frontend",
   "version": "0.1.0",
-  "lockfileVersion": 1,
```

Comment:
let's remove the changes to package lock

Reply:
done